### ISSUE TITLE
Docs won't start/Improve contributor workflow

### ISSUE THREAD

**Issue by rosskevin** [dmtrKovalenko/material-ui-pickers#168]
## Environment

| Tech                | Version |
|---------------------|---------|
| material-ui-pickers |    `master`    08891717ffa83748536a6cbd40dd2e32a5de7039 |

## Steps to reproduce
```
cd lib
npm install
npm start
```

## Expected behavior
Docs site starts

## Actual behavior
```
Failed to compile.

./src/Demo/Examples/BasicUsage.jsx
Module not found: Can't resolve 'material-ui-pickers' in '/Users/kross/projects/material-ui-pickers/docs/src/Demo/Examples'
```

## Context
I'm trying to PR some tests/fixes for typescript, and wanted to run the docs site to debug/check types and can't start it.

**Comment by cherniavskii** [comment]
You need to link `material-ui-pickers` package:
https://github.com/dmtrKovalenko/material-ui-pickers/blob/master/docs/README.md

I had same problem when started project locally for the first time, so I think it's time to describe this flow in one place - for example CONTRIBUTING.md

**Comment by rosskevin** [comment]
Thanks @cherniavskii 
Along the same lines of developer workflow with this library:
- you run the docs with a built lib version (no sourcemaps)
- you cannot run a minimal demo of lib with sourcemaps

This makes it significantly more difficult to use the docs as a playground to understand and contribute to the library.  I really need a source mapped runtime if I want any chance of figuring out the mystery of #169

**Comment by dmtrKovalenko** [comment]
@rosskevin no. We should use symlinks to have an ability to test the libriry in most production build. For this purpose there is a readme in the docs folder

**Comment by rosskevin** [comment]
You don't think we should be able to run any examples with sourcemaps for development/debug purposes?  If so, that sounds like a really painful developer workflow and it will slow development and decrease contributors.

**Comment by sakulstra** [comment]
I just faced the same problems. Can't we just import the lib relatively in docs?, then the docs would automatically reflect the changes we're working on.
not strictly related, but for me the tests throw a lot of warnings.

**Comment by dmtrKovalenko** [comment]
Fixed in 1.0.0-rc.2
